marklogic-data-hub-central/ui/src/pages/LoadData.test.tsx

```diff
@@ -0,0 +1,84 @@
+import React, {useContext} from 'react';
+import { render, fireEvent, waitForElement, cleanup } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import TilesView from './TilesView';
+import {AuthoritiesContext, AuthoritiesService} from '../util/authorities';
+import axiosMock from 'axios';
+import curateData from '../config/bench.config';
+import authorities from '../config/authorities.config';
+import data from "../config/bench.config";
+import LoadData from "./LoadData";
+
+jest.mock('axios');
+
+describe('LoadData component', () => {
+  test('Verify LoadData component cannot edit with only readIngestion authority', async () => {
+      const authorityService = new AuthoritiesService();
+      authorityService.setAuthorities(['readIngestion']);
+
+      await curateData.setupMockAPIs(axiosMock);
+
+      const { getByText, getByTitle, getByLabelText, getByTestId } = render(<AuthoritiesContext.Provider value={authorityService}><LoadData/></AuthoritiesContext.Provider>);
+
+      expect(await(waitForElement(() => getByTitle('table')))).toBeInTheDocument();
+
+      // Check for steps to be populated
+      expect(axiosMock.get).toBeCalledWith('/api/steps/ingestion');
+      expect(getByText('loadXML')).toBeInTheDocument();
+
+      // Check table layout
+      const tableMenuItem = getByTitle('table');
+      expect(tableMenuItem).toBeInTheDocument();
+      await fireEvent.click(tableMenuItem);
+
+      await fireEvent.click(getByTestId('loadXML-settings'));
+      expect(await(waitForElement(() => getByText('Target Database:')))).toBeInTheDocument();
+
+      expect(getByText('Save')).toBeDisabled();
```

Comment:
just like update here, can we add assertions for create(Add New) and delete as well for both readIngestion and writeIngestion.

Reply:
@ryanjdew this is the only missing scenario in the tests. Approving the PR. Have noted this in the test plan as well. Maybe we can cover this when writing tests for hc-load-writer